### PR TITLE
Map colors update

### PR DESCRIPTION
**Pull request by NateBaldwinDesign**
* Updated color scale used in choropleth visualization
* Updated method of selecting colors from scale by way of d3 color & scale
* Updated tangent function for normalizing data for color scales

Screen Shot 2020-03-19 at 12 57 58 PM

Screen Shot 2020-03-19 at 12 58 10 PM

### CODE REVIEW

File: site/index.js [hunk 1]
```diff
@@ -8,24 +8,54 @@
...
+const noCasesColor = '#faffef';
+const noPopulationDataColor = '#ffffff';
 
 const outlineColorHighlight = 'rgb(0,0,0)';
-const outlineColor = 'rgba(175, 175, 175, 0.5)';
+const outlineColor = 'rgba(0, 0, 0, 0.3)';
 
 const choroplethColors = {
-  stoplight: ['#eeffcd', '#b4ffa5', '#ffff00', '#ff7f00', '#ff0000'],
+  stoplight: ['#faffef', '#f3fac1', '#f6f191', '#ffe15d', '#fec327', '#ff9b00', '#fe7000', '#fa4d13', '#c52155', '#842e79'],
```
Comment (lazd) [inline site/index.js]:
```suggestion
  stoplight: ['#eeffcd', '#b4ffa5', '#ffff00', '#ff7f00', '#ff0000'],
  brownish: ['#faffef', '#f3fac1', '#f6f191', '#ffe15d', '#fec327', '#ff9b00', '#fe7000', '#fa4d13', '#c52155', '#842e79'],
```

File: site/index.js [hunk 2]
```diff
@@ -8,24 +8,54 @@
...
+}
+function normalizePercent(min, max, input) {
+  const range = max - min;
+  const correctedStartValue = input - min;
+  const percentage = (correctedStartValue * 100) / range;
+  return percentage / 100;
+}
+
+let domainArray = [];
+const colorsArray = choroplethColors.stoplight;
```
Comment (lazd) [inline site/index.js]:
```suggestion
const colorsArray = choroplethColors[choroplethColor];
```

File: site/index.js [hunk 3]
```diff
@@ -8,24 +8,54 @@
...
+  stoplight: ['#faffef', '#f3fac1', '#f6f191', '#ffe15d', '#fec327', '#ff9b00', '#fe7000', '#fa4d13', '#c52155', '#842e79'],
   heat: ['#FFFFFF', '#ffff5e', '#ffe70c', '#fead0a', '#fd6f08', '#fd2907', '#fd0407'],
   peach: ['rgb(253,222,166)', 'rgb(255,188,134)', 'rgb(249,152,133)', 'rgb(232,110,129)', 'rgb(224,88,136)'],
   pink: ['rgb(255, 244, 221)', 'rgb(255, 221, 215)', 'rgb(255, 197, 210)', 'rgb(254, 174, 203)', 'rgb(250, 150, 196)', 'rgb(245, 126, 189)', 'rgb(239, 100, 181)', 'rgb(232, 70, 173)', 'rgb(210, 56, 161)', 'rgb(187, 46, 150)', 'rgb(163, 36, 140)', 'rgb(138, 27, 131)', 'rgb(113, 22, 124)', 'rgb(86, 15, 116)', 'rgb(55, 11, 110)', 'rgb(0, 9, 104)'],
   viridis: ['#fde725', '#d8e219', '#addc30', '#84d44b', '#5ec962', '#3fbc73', '#28ae80', '#1fa088', '#21918c', '#26828e', '#2c728e', '#33638d', '#3b528b', '#424086', '#472d7b', '#48186a'],
   magma: ['#fcfdbf', '#fde2a3', '#fec488', '#fea772', '#fc8961', '#f56b5c', '#e75263', '#d0416f', '#b73779', '#9c2e7f', '#832681', '#6a1c81', '#51127c', '#36106b', '#1d1147', '#0a0822']
 };
 
-const choroplethStyle = 'pureRatio';
+function returnLightness(c) {
```
Comment (lazd) [inline site/index.js]:
```suggestion
const choroplethColor = 'brownish';

function returnLightness(c) {
```

**Comment by NateBaldwinDesign** [comment]
@lazd applied your suggestions and updated the name of my colorscale.

**Pull request merged**